lib/processRunner.js

```diff
@@ -20,52 +20,23 @@ var spawn = require('child_process').spawn
 var execSync = require('child_process').execSync
 var psTree = require('ps-tree')
 var winPsTree = require('winpstree')
-var fs = require('fs')
-var path = require('path')
-var dotenv = require('dotenv')
+
 
 module.exports = function () {
   var isWin = /^win/.test(process.platform)
 
-
-  var handleIpAddress = function (container, cmd) {
-    var ipAddress
-
-    ipAddress = container.specific.ipAddress || '127.0.0.1'
-    cmd = cmd.replace(/__TARGETIP__/g, ipAddress)
-    return cmd
-  }
-
-
-
-  var generateEnvironment = function (containerEnv) {
-    var envArgs = isWin ? ' ' : 'env '
-    _.each(_.keys(containerEnv), function (key) {
-      if (isWin) {
-        envArgs += 'set ' + key + '=' + containerEnv[key] + '&&'
-      } else {
-        envArgs += '"' + key + '=' + containerEnv[key] + '" '
-      }
-    })
-    return envArgs
-  }
-
-
-
   var run = function (mode, container, exitCb, cb) {
-    var cmd = container.specific.execute.exec
-    var cwd = container.specific.path
-    var envFile = getEnvFile(container)
-
+    var cmd = container.run
+    var cwd = container.path
     var env
-    var toExec
     var child = {}
-    var envArgs = ''
     var called = false
 
-    if (cmd.slice(0, 5) === 'node ' && cmd.slice(0, 7) !== 'node -r') {
-      // use same node running fuge
-      // and quote windows path to handle potential spaces
+    console.log('running: ' + container.run)
```

Comment:
no - expected in the fuge output at the moment - although I agree should probably be moved to the main fuge module